src/puppetlabs/puppetdb/http/nodes.clj

```diff
@@ -13,14 +13,12 @@
 
 (defn node-status
   "Produce a response body for a single environment."
-  [api-version node db url-prefix]
+  [api-version node options]
   (let [status (first
-                (eng/stream-query-result :nodes
-                                         api-version
-                                         ["=" "certname" node]
+                (eng/stream-query-result api-version
+                                         ["from" "nodes" ["=" "certname" node]]
```

Comment:
Just making a note that anything that runs `stream-query-result` directly might not handle database streaming errors correctly, and also won't pretty-print.